src/buildercore/bigquery.py

```diff
@@ -1,4 +0,0 @@
-BIGQUERY_SCHEMAS_FOLDER = 'src/buildercore/bigquery/schemas'
```

Comment:
like `buildercore.fastly`, this module is meant to store BigQuery-specific code that is not Terraform-specific (and hence would go into `buildercore.terraform`). Let's see what emerges

Reply:
If there was any significant logic in there it would have been preserved. Should the BigQuery wrangling get large we can always re-create it

Reply:
Given the delegation to Terraform of most of the file-loading stuff, this module should not be needed for now